### DISCUSSION TITLE
Gatekeeper2 UDP port

### DISCUSSION THREAD

**Discussion by Solamain** [k8thekat/GatekeeperV2#91] [Q&A]
In AMP the UDP port is red, showing it is not connected.  What should I be connecting it to?

**Comment by k8thekat** [comment]
@Solamain 

I am honestly unsure. From my understanding of it AMP was assigning all the proper ports itself.

**Comment by Solamain** [comment]
![image](https://github.com/k8thekat/GatekeeperV2/assets/38622677/dd22a1cd-9a69-40b8-9a14-3069aa4deab1)
as a visual

**Reply by k8thekat** [reply]
I am not sure it needs to even use that UDP port. 

Does the bot work just fine? If so I would ignore it.

**Reply by Solamain** [reply]
It stops reporting after a short while, and even restarting it does not always work.

**Reply by k8thekat** [reply]
What version of AMP?

I will attempt to test this locally ASAP.

**Reply by Solamain** [reply]
Apologies, we have decided to abandon AMP as we were getting excessive lag spikes while using it to host one ark server as a test, while having a full cluster up before using docker-compose and manually configuring the settings for all servers and having 2x the mods produced no lag spikes besides the short ones from world saves.

**Reply by k8thekat** [reply]
Okay, I do apologize about the UDP issue. I will look into that for future releases.

Best of luck!
